src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -74,6 +75,13 @@ class Task extends React.Component<Props> {
         const hiddenAssigneeCount = assigneeCount - MAX_AVATARS;
         const isOverdue = due_at ? status === TASK_NEW_INCOMPLETE && new Date(due_at) < Date.now() : false;
 
+        const shouldShowActions =
+            currentUserAssignment &&
+            currentUserAssignment.permissions &&
+            currentUserAssignment.permissions.can_update &&
```

Comment:
Looks like this could benefit from `lodash/get`.

Reply:
Yeah, except get mangles Flow, so you can't validate the object you are parsing is valid.

Reply:
Would the following work?

```
const canUpdate: boolean = getProp(currentUserAssignment, 'permissions.can_update', false);
```

Reply:
Yes, but Flow won't complain if you put in e.g., `permissions.can_kjhasdf` into `_.get`, but I want it to.